pallets/spaces/src/benchmarking.rs

```diff
@@ -0,0 +1,87 @@
+//! Spaces pallet benchmarking.
+
+use super::*;
+
+use frame_benchmarking::benchmarks;
+use frame_benchmarking::whitelisted_caller;
+use frame_support::ensure;
+use frame_support::pallet_prelude::Get;
+use frame_system::RawOrigin;
+use sp_std::vec;
+
+use crate::Config;
+use crate::types::*;
+
+use pallet_permissions::{
+    default_permissions::DefaultSpacePermissions,
+    SpacePermissionSet,
+};
+
+
+fn dummy_space_content() -> Content {
+    Content::IPFS(b"QmRAQB6YaCaidP37UdDnjFY5aQuiBrbqdyoW1CaDgwxkD4".to_vec())
+}
+
+fn get_next_space_id<T: Config>() -> SpaceId {
+    NextSpaceId::<T>::get()
+}
+
+fn get_new_space_id<T: Config>() -> SpaceId {
+    let space_id = get_next_space_id::<T>();
+    NextSpaceId::<T>::mutate(|n| *n += 1);
+    space_id
+}
+
+fn create_dummy_space<T: Config>() -> Space<T> {
+    let id = get_new_space_id::<T>();
+    let space = Space::new(
+        id,
+        None,
+        whitelisted_caller::<T::AccountId>(),
+        Content::None,
+        None,
+    );
+    SpaceById::<T>::insert(id, space);
+
+    SpaceById::<T>::get(id).expect("Expected space to exist")
```

Comment:
We used to do like this in our code. Like parity team does actually...

```suggestion
    SpaceById::<T>::get(id).expect("qed; space should exist")
```